web/src/integTest/java/com/microsoft/applicationinsights/test/utils/PropertiesUtils.java

```diff
@@ -1,44 +0,0 @@
-package com.microsoft.applicationinsights.test.utils;
```

Comment:
It would be better to keep this class so it can be used both for the environment property file and for the test specific property file